compiler/rustc_mir_build/src/build/custom/parse/instruction.rs

```diff
@@ -275,7 +275,8 @@ impl<'tcx, 'body> ParseCtxt<'tcx, 'body> {
                     fields.iter().map(|e| self.parse_operand(*e)).collect::<Result<_, _>>()?
                 ))
             },
-            ExprKind::Adt(box AdtExpr{ adt_def, variant_index, args, fields, .. }) => {
+            ExprKind::Adt(box AdtExpr { adt_def, variant_index, args, fields, .. }) => {
+                // HERE?
```

Comment:
What is the question?

Reply:
It was a placeholder for myself while untangling things before I got it all working.